x/trace/trace.go

```diff
@@ -77,3 +74,18 @@ func setupTracer(logger log.Logger, cfg jaegercfg.Configuration) (opentracing.Tr
 func GlobalTracer() opentracing.Tracer {
 	return opentracing.GlobalTracer()
 }
+
+var _ jaeger.Logger = (*jaegerLogger)(nil)
+
+// adapter for jaeger.Logger
+type jaegerLogger struct {
```

Comment:
Jaeger comes with a go-kit adapter: "github.com/jaegertracing/jaeger-lib/client/log/go-kit"
But we need to write our own here.

Reply:
Maybe I should move this to `base/logger` eventually?